lua/autorun/sh_vkx_scp_096.lua

```diff
@@ -152,6 +152,16 @@ hook.Add( "guthscpbase:config", "vkxscp096", function()
 						desc = "The Head Bone's Name to use for detecting obstacles between the potential target & SCP-096",
 						default = "ValveBiped.Bip01_Head1",
 					},
+					{
+						type = "NumWang",
+						name = "Detection angle",
+						id = "detection_angle",
+						desc = "Cosinus of the angle between the vector ply_heads_to_096_heads and ply_aim_vector when 096 is triggered. 096 detect player when angle between the vector ply_heads_to_096_heads and ply_aim_vector is greater than this cosinus.",
```

Comment:
The description is too much technical, I propose this one instead:
"Detection Method must be set to 'Serverside'! Cosine of the victim's field of view angle used to trigger SCP-096. Increasing this value will make the detection threshold smaller and vice-versa. By default, set to 0.55 which is equivalent to an angle of 56°."